test/test_jit.py

```diff
@@ -11877,18 +11874,23 @@ def test_lstm_cuda(self):
         self.assertGraphContainsExactly(
             forward_graph, 'prim::FusionGroup', 1, consider_subgraphs=True)
         self.assertExpectedGraph(forward_graph, subname='forward')
+        self.assertTrue(len(forward_graph.nodes()) == 2)
+        FileCheck().check("DifferentiableGraph").check_next("TupleConstruct") \
```

Comment:
It's a little hard to read this line. I think what we're checking here is that LSTM forward is completely differentiable, except for TupleConstruct. Maybe add a comment? (maybe we should make TupleConstruct differentiable, but that's another discussion).